cmd/server/assets/header.html

```diff
@@ -208,5 +209,28 @@ <h6 class="dropdown-header">Actions</h6>
       $target.val(value);
     })
   });
+
+  var $main = $('main');
+
+  function clearExistingAlerts() {
+    $main.find('div.alert').remove();
+  }
+
+  function alert(message, errLevel) {
```

Comment:
`alert` is a javascript built-in technically it's window.alert, but it also exists in the default namespace. Maybe name this `flash`?

Reply:
done